### ISSUE TITLE
Can't add device on homey

### ISSUE THREAD

**Issue by Schippelkopf** [tibbercom/com.tibber.athom#13]
https://user-images.githubusercontent.com/88089657/127314501-3fe43715-c612-4bb4-9c89-343e01d08641.mp4

**Comment by toini** [comment]
Hi, looks like your subscription is currently not running but instead has status "awaiting market". That might cause issues. I'll double check.

**Comment by toini** [comment]
Yes, the homey app requires that there is a running subscription before allowing home to be paired. https://github.com/tibbercom/com.tibber.athom/blob/master/drivers/home/driver.js#L23

We will need to confirm if it could be adjusted to allow "awaiting market", too.

**Comment by lovethyresson** [comment]
Adding to known issues for now.
